### DISCUSSION TITLE
TypeError: (0 , import_react14.createContext) is not a function using NextJS13

### DISCUSSION THREAD

**Discussion by xdmarttt** [tailwindlabs/headlessui#1972] [Help] [answered]
I created a new project using Next13 + React18.

Unfortunately, I encountered this problem:
![image](https://user-images.githubusercontent.com/29189082/198899736-ba56a7b0-82f4-474b-a2e9-9cbf1d3a4206.png)

I tried some possible solution that can here https://github.com/tailwindlabs/headlessui/issues/681 but still not working.

**Comment by calebpdx** [comment, marked as answer]
> I created a new project using Next13 + React18.
> 
> 
> 
> Unfortunately, I encountered this problem:
> 
> ![image](https://user-images.githubusercontent.com/29189082/198899736-ba56a7b0-82f4-474b-a2e9-9cbf1d3a4206.png)
> 
> 
> 
> I tried some possible solution that can here https://github.com/tailwindlabs/headlessui/issues/681 but still not working.
> 
> 

You have to make it a client component for it to work within the app directory.

Add "use client";

To the first line of your component's file and you'll be up and running in no time!

**Reply by dylpckl** [reply]
thank you so much!
